src/jdk.jlink/share/classes/jdk/tools/jlink/resources/plugins.properties

```
@@ -59,38 +59,20 @@ Class optimization: convert Class.forName calls to constant loads.
 class-for-name.usage=\
 \  --class-for-name          Class optimization: convert Class.forName calls to constant loads.
 
-compress.argument=<0|1|2>[:filter=<pattern-list>]
+compress.argument=<zip-[0-9]>[:filter=<pattern-list>]
 
-compress.description=\
-Compress all resources in the output image.\n\
-Level 0: No compression\n\
-Level 1: Constant string sharing\n\
-Level 2: ZIP.\n\
-An optional <pattern-list> filter can be specified to list the pattern of\n\
-files to be included.
+compress.description= Compression to use in compressing resources.
 
 compress.usage=\
-\  --compress <0|1|2>[:filter=<pattern-list>]\n\
-\                            Compress all resources in the output image.\n\
-\                            Level 0: No compression\n\
-\                            Level 1: Constant string sharing\n\
-\                            Level 2: ZIP.\n\
-\                            An optional <pattern-list> filter can be\n\
-\                            specified to list the pattern of \n\
-\                            files to be included.
-
-compact-cp.argument=<resource paths>
-
-compact-cp.description=Constant Pool strings sharing.\n\
-By default, all resources are compressed. You can express the set \n\
-of resources to compress or not compress (use ^ for negation).
-
-compact-cp.usage=\
-\ --compact-cp <resource paths>\n\
-\                             Constant Pool strings sharing.\n\
-\                             By default, all resources are compressed.\n\
-\                             You can express the set of resources to\n\
-\                             compress or not compress (use ^ for negation).
+\  --compress <compress>     Compression to use in compressing resources:\n\
+\                            Accepted values are:\n\
+\                            zip-[0-9], where zip-0 provides no compression,\n\
+\                            and zip-9 provides the best compression.\n\
+\                            Default is zip-6.
+
+compress.warn.argumentdeprecated=\
+Warning: The {0} argument for --compress is deprecated and may be removed from a future release
```

```suggestion
Warning: The {0} argument for --compress is deprecated and may be removed in a future release
```

Sorry missed this in the previous round.  `s/from/in`

---

Got it!